tests/installation/scc_registration.pm

```diff
@@ -20,6 +20,16 @@ use testapi;
 use registration;
 
 sub run() {
+    if (get_var("SCC_EXPECT_ERROR")) {
+        while (1) {
+            if (check_screen 'registration-error') {
+                send_key 'alt-o';
+                sleep 2;
```

Comment:
I don't like sleep's. Can you replace this? e.g. delete if not necessary, replace by wait_still_screen, assert_screen, wait_serial, …